src/docs/arch-storage-extension.dox

```diff
@@ -0,0 +1,104 @@
+/*! @arch_page arch-cloud-storage-extension Storage Source
+
+@section extension_storage_source Cloud Storage Source Implementation
+
+The WiredTiger extension allows the control of how files are stored via the storage source
+abstraction layer. More information can be found through @ref custom_storage_sources. There are
+three storage sources that extends the WiredTiger's capability to store the data files onto the
```

Comment:
Grammar: "that extends the WiredTiger's capability to store the data files onto the cloud storage" -> "that extend WiredTiger's capability to store data files into cloud storage"

Reply:
done!

Reply:
I still don't think this is quite right, there's some unnecessary `the`s floating around (I fixed a few things in my above suggestion, sorry if that was unclear.)